index.js

```diff
@@ -10,7 +10,7 @@ try {
   const product = input1 * input2;
 
   // Create a custom message
-  const message = `The inputs are ${input1} and ${input2}. Sum is ${sum}, product is ${product}.`;
```

Comment:
There appears to be a typo in the variable name `awesome-product`. Variable names in JavaScript should not contain hyphens. Please use camelCase or snake_case naming conventions instead.